content/Registration and Authentication in your Django app with dj-rest-auth part1- Bruno Michetti.md

```diff
@@ -0,0 +1,215 @@
+## Registration and Authentication in your Django app with dj-rest-auth (part 1)
+
+A huge amount of existing applications have registration and authentication for users. Maybe every developer in the world has implemented something related to this in their work or while they learned. After the creation of the [Django REST framework](https://www.django-rest-framework.org/), Django developers started to implement more and more app-level REST API endpoints.  
+As a result, an open source package called [dj-rest-auth](https://dj-rest-auth.readthedocs.io/en/latest/index.html) has been developed to provide a set of REST API endpoints to handle user registration and authentication tasks. In this article divided into two parts, I will talk about this package; and given that it can generate some problems at first, I will give you some tips to avoid them.
```

Comment:
```suggestion
As a result, an open-source package called [dj-rest-auth](https://dj-rest-auth.readthedocs.io/en/latest/index.html) has been developed to provide a set of REST API endpoints to handle user registration and authentication tasks. In this article, I will not only talk about this particular package but also, I will give you some tips to avoid some of the most common problems faced when configuring it.
```

Reply:
I'll accept this change but I will the fact that is divided into two parts